pkg/model/resource/resource.go

```diff
@@ -43,29 +44,41 @@ type Resource struct {
 	// ImportAlias is a cleaned concatenation of Group and Version.
 	ImportAlias string `json:"-"`
 
-	// Package is the go package of the Resource.
-	Package string `json:"package,omitempty"`
+	// Endpoint is the API endpoint which is used import the go package of the Resource.
+	Endpoint string `json:"endpoint,omitempty"`
```

Comment:
I don't understand why this is called `endpoint` now, and is more confusing than `path` or `package`.

Reply:
Because `k8s.io/api/apps/v1` is a Rest API endpoint.

Reply:
1. This value is not related to the REST endpoint in this context, it is related to the native type import path. In any case why would a `Resource` ever need to know about a REST endpoint?
1. k8s REST endpoints have a [different structure](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups-and-versioning). Ex. import `k8s.io/api/apps/v1` might have a REST endpoint `/apis/apps/v1`, and `k8s.io/api/rbac/v1` might have `/apis/rbac.authorization.k8s.io/v1`.

Reply:
You are definitely right here. Good point. So, yes. I would agree that path is a better name either. 👍